core/src/main/scala/kafka/cluster/Replica.scala

```diff
@@ -35,6 +35,13 @@ class Replica(val brokerId: Int,
   // the log end offset value, kept in all replicas;
   // for local replica it is the log's end offset, for remote replicas its value is only updated by follower fetch
   @volatile private[this] var logEndOffsetMetadata = LogOffsetMetadata.UnknownOffsetMetadata
+  // the log start offset value, kept in all replicas;
+  // for local replica it is the log's start offset, for remote replicas its value is only updated by follower fetch
+  @volatile private[this] var _logStartOffset = 0L
```

Comment:
I used `_*` here because we have methods `lastCaughtUpTimeMs`, `logStartOffset` and `lowWatermark`. What would you suggest to name these variables?